tests/test_alma.py

```diff
@@ -0,0 +1,195 @@
+import datetime
+import urllib.parse
+
+import requests_mock
+
+
+def test_client_initializes_with_expected_values(alma_client):
+    assert alma_client.base_url == "https://example.com"
+    assert alma_client.headers == {
+        "Authorization": "apikey just-for-testing",
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+    }
+    assert alma_client.timeout == 10
+
+
+# sample_data
+def test_create_invoice(alma_client):
+    test_url = "https://example.com/acq/invoices"
+    test_payload = {"test": "invoice_data"}
+    mocked_response = {"json": "invoice_response"}
+    with requests_mock.Mocker() as mocker:
```

Comment:
I would definitely create a `mock_alma` fixture like we had in the original repo's `conftest.py`. I know it can be tricky to get that to work at times but I'm happy to help if needed and it will make your life so much easier down the road

Reply:
I think it is a lot more clear when testing the alma module to do the mocking directly in the tests. I agree that when testing some of the downstream modules it makes sense to have the mocked responses in a fixture.

Reply:
Yeah, it also keeps the test concerns separate. Alma unit test fixtures don't depend on SAP unit and integration test fixtures, and vice versa. I like the approach.